src/DynamoCoreWpf/UI/Themes/Modern/SidebarGridStyleDictionary.xaml

```diff
@@ -0,0 +1,220 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+    <!-- Region Control Templates-->
+
+
+    <!-- Region LibraryView + LibrarySearchView-->
+
+    <ControlTemplate x:Key="LibraryThumbTemplate"
+                     TargetType="{x:Type Thumb}">
+        <Grid>
+            <Rectangle HorizontalAlignment="Stretch"
+                       VerticalAlignment="Stretch"
+                       Width="Auto"
+                       Height="Auto"
+                       Fill="Transparent" />
+            <Border x:Name="Rectangle1"
+                    HorizontalAlignment="Stretch"
+                    VerticalAlignment="Stretch"
+                    Width="7"
+                    Height="Auto"
+                    Background="#aaaaaa"
+                    Opacity="0.5"
+                    CornerRadius="2" />
+        </Grid>
+    </ControlTemplate>
+
+    <ControlTemplate x:Key="LibraryScrollBarTemplate"
+                     TargetType="{x:Type ScrollBar}">
+        <Grid x:Name="Bg"
+              SnapsToDevicePixels="true"
+              Background="Transparent">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="0.00001*" />
+            </Grid.RowDefinitions>
+            <Track x:Name="PART_Track"
+                   IsDirectionReversed="true"
+                   IsEnabled="{TemplateBinding IsMouseOver}">
+                <Track.Thumb>
+                    <Thumb Style="{DynamicResource LibraryVerticalThumb}" />
+                </Track.Thumb>
+            </Track>
+        </Grid>
+    </ControlTemplate>
+
+    <ControlTemplate x:Key="LibraryScrollViewerControlTemplate"
```

Comment:
`LibraryScrollViewerControlTemplate` and `TooltipScrollViewerControlTemplate` are mostly the same, except `Width` and `Style`. I know that `Style` cannot be unified in this case (due to difference in their behaviors), but can we specify the `Style` and `Width` separately in the place where the template is used? Or is that not possible?